gymnasium/core.py

```diff
@@ -445,7 +441,7 @@ def action(self, act):
                 return self.disc_to_cont[act]
 
         if __name__ == "__main__":
-            env = gym.make("LunarLanderContinuous-v2")
```

Comment:
The general recommended usage is `import gymnasium as gym`, so I don't think it's helpful to expand it in the examples since that's not the code that will generally appear anywhere